frontend/src/stores/SongsStore.js

```diff
@@ -1,8 +1,11 @@
 import { Store } from 'flux/utils';
 import ApiService from '@Services/ApiService.js';
 import AppDispatcher from '@Services/AppDispatcher.js';
-import { updateSongsAction, updateSongAction, updateErrorAction, SongActions } from '@Actions/SongActions.js';
```

Comment:
Deze acties worden nog gebruikt in de Store en zijn nodig voor de nummers pagina :-)